src/Tribe/RSVP.php

```diff
@@ -61,6 +61,13 @@ class Tribe__Tickets__RSVP extends Tribe__Tickets__Tickets {
 	public $security_code = '_tribe_rsvp_security_code';
 
 	/**
+	 * Meta key that if this attendee wants to show on the attendee list
+	 *
+	 * @var string
+	 */
+	public $optout = '_tribe_rsvp_attendee_optout';
```

Comment:
Sorry one more - if we change optout to opt_out here, let's do it everywhere! (Unless there's a back compat concern of course)